iron-overlay-manager.html

```diff
@@ -78,21 +78,6 @@
     this._minimumZ = Math.max(this._minimumZ, minimumZ);
   };
 
-  Polymer.IronOverlayManagerClass.prototype.focusOverlay = function() {
```

Comment:
this is not needed anymore since now each overlay keeps track of the lastFocusedNode before applying the focus.